### PR TITLE
add clash-for-windows

### PR DESCRIPTION
**Pull request by specter119**
add clash-for-windows

### CODE REVIEW

File: bucket/clash-for-windows.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "shortcuts": [
+        [
+            "Clash for Windows.exe",
+            "Clash for Windows"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/Fndroid/clash_for_windows_pkg/releases/download/$version/Clash.for.Windows-$version-win.zip",
+        "hash": {
```
[the remark this answers is not in the document]
Comment (specter119) [reply bucket/clash-for-windows.json]: @Ash258 I have convert the hash to "$url.sha256" according to [Add hash to autoupdate](https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate#add-hash-to-autoupdate).

[the remark this answers is not in the document]
Comment (Ash258) [reply bucket/clash-for-windows.json]: There is no `sha256` file.
There is only Clash for Wnidows setup.exe and win.zip.

Delete hash property.

**Comment by r15ch13** [comment]
Is there a reason why it now has only an empty `README.md`?

**Comment by specter119** [comment]
@r15ch13 The clash related projects will remain and continue developing for a long time, according to their [telegram channel](https://t.me/clash_discuss). This PR can be closed if you mind the odd READMEs. The develops may have some reasons to do this. Also I have update the homepage for clash for windows, but it's written in Chinese.

File: bucket/clash-for-windows.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (linsui) [reply bucket/clash-for-windows.json]: Now there is a hash file.

File: bucket/clash-for-windows.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "shortcuts": [
+        [
+            "Clash for Windows.exe",
+            "Clash for Windows"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/Fndroid/clash_for_windows_pkg/releases/download/$version/Clash.for.Windows-$version-win.7z",
+        "hash": "$baseurl/sha256sum"
```
Comment (Ash258) [inline bucket/clash-for-windows.json]: ...

```suggestion
        "hash": {
            "url": "$baseurl/sha256sum",
            "regex": "7z:\\s+$sha256"
        }
```

File: bucket/clash-for-windows.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+    "homepage": "https://docs.cfw.lbyczf.com",
+    "description": "A Windows GUI based on Clash.",
+    "license": "custom",
```
Comment (linsui) [inline bucket/clash-for-windows.json]: If there is no license, please use `Unknown` . Or provide a url.

File: bucket/clash-for-windows.json (hunk 2 above)
Comment (specter119) [reply bucket/clash-for-windows.json]: the latest version have received this change.

File: bucket/clash-for-windows.json (hunk 3 above)
Comment (specter119) [reply bucket/clash-for-windows.json]: thank, I have change the license  to "Unknown"

**Comment by specter119** [comment]
@Ash258 I don't know why there still remains a change request, when I resolved all change request.

**Comment by Ash258** [comment]
- https://github.com/lukesampson/scoop-extras/pull/2131#issuecomment-493116708
- Chinese homepage

I cannot approve it.

**Comment by specter119** [comment]
@Ash258 I will contact the developer, to see if he will provide an English homepage recently.

**Comment by specter119** [comment]
close for missing English homepage and blank github readme.

**Comment by linsui** [comment]
FYI, https://github.com/h404bi/dorado/blob/master/bucket/clash-for-windows.json

**Comment by specter119** [comment]
@linsui Thanks, but what for? I need updating clash-for-windows rather than just installing. The repos maintained personally still need update his buckets. I have found several personal repos contains clash-for-windows.
If I have time, I will add an English translate to the homepage for clash for windows. And then add it to scoop-extras, again.
Anyway, thanks in advance.

**Comment by linsui** [comment]
If you just want to update manifest automatically, maybe this is useful. https://github.com/Ash258/Scoop-GithubActions

**Pull request closed without merging**